extension/src/popup/views/SignTransaction/index.tsx

```diff
@@ -17,35 +14,22 @@ import { decodeMemo } from "popup/helpers/decodeMemo";
 import { rejectTransaction, signTransaction } from "popup/ducks/access";
 import { settingsNetworkDetailsSelector } from "popup/ducks/settings";
 
-import { Button } from "popup/basics/Buttons";
-import { ButtonContainer, SubmitButton } from "popup/basics/Modal";
+import {
+  ButtonContainer,
+  SubmitButton,
+  ModalWrapper,
+} from "popup/basics/Modal";
 
 import { METRIC_NAMES } from "popup/constants/metricsNames";
 
 import { FirstTimeWarningMessage } from "popup/components/warningMessages/FirstTimeWarningMessage";
-import { Header } from "popup/components/Header";
 import { FlaggedWarningMessage } from "popup/components/warningMessages/FlaggedWarningMessage";
+import { ModalInfo } from "popup/components/ModalInfo";
 import { WarningMessage } from "popup/components/WarningMessage";
-import { PunycodedDomain } from "popup/components/PunycodedDomain";
 import { Transaction } from "popup/components/signTransaction/Transaction";
 import { TransactionHeader } from "popup/components/signTransaction/TransactionHeader";
 
-const El = styled.div`
-  padding: 1.5rem 1.875rem;
-  box-sizing: border-box;
-`;
-const HeaderEl = styled.h1`
-  color: ${COLOR_PALETTE.primary};
-  font-weight: ${FONT_WEIGHT.light};
-  margin: 0;
-`;
-
-const SubheaderEl = styled.h3`
-  font-weight: ${FONT_WEIGHT.bold};
-  font-size: 0.95rem;
-  letter-spacing: 0.1px;
-  color: ${COLOR_PALETTE.primary};
-`;
```

Comment:
4) As I'm creating new styled elements, I'm deleting the unused styled-componenets